CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+### Mapbox Navigation SDK 1.2.0-alpha.3 - October 21, 2020
+
+Visit our [1.0.0 migration guide](https://github.com/mapbox/mapbox-navigation-android/wiki/1.0-Navigation-SDK-Migration-Guide) and the [documentation pages](http://docs.mapbox.com/android/navigation/overview) for more information.
+
+### Changelog
+
+#### Features
+* Update the algorithm to calculate the vanishing point of the route to use EPSG:3857 projection instead of turf measurements. The precision is now spot on, even on extremely long routes, it even greatly outperforms the precision of the over zoomed map tiles. It also improves the performance of the `MapRouteLine` [#3661](https://github.com/mapbox/mapbox-navigation-android/pull/3661)
+
+#### Bug fixes
+* Assign `currentHorizon`, `currentType` and `currentPosition` from the main thread to avoid synchronization issues [#3653](https://github.com/mapbox/mapbox-navigation-android/pull/3653)
+* Keep the selected route as primary after map style has been changed [#3664](https://github.com/mapbox/mapbox-navigation-android/pull/3664)
+* Don't obfuscate `ReplayDataSource` to prevent a crash in `ReplayHistoryActivity` [#3646](https://github.com/mapbox/mapbox-navigation-android/pull/3646)
+
+#### Other changes
+* Bump mapboxCommonNative version to 7.1.1 [#3650](https://github.com/mapbox/mapbox-navigation-android/pull/3650)
```

Comment:
Let's remove this note, we're listing the version at the top already.

Reply:
done